addthis.module

```diff
@@ -80,6 +80,23 @@ function addthis_menu() {
     'file' => AddThis::ADMIN_INCLUDE_FILE,
     'weight' => 10
   );
+
+  // Test pages.
```

Comment:
I don't know if the test pages that I build should be here or in another addthis_test module. Suggestions?